exercises/difference-of-squares/example.exs

```diff
@@ -1,6 +1,6 @@
 defmodule Squares do
   @moduledoc """
-  Calculate sum of squares, square of sums, difference between two sums from 1 to a given end number.
+  Calculate sum of squares, square of sum, difference between two sum from 1 to a given end number.
```

Comment:
This should still be `difference between two sums from...`

Reply:
Thank you for your remark. I'll change it now.